fastfood/food.py

```diff
@@ -42,33 +41,23 @@ def build_cookbook(build_config, templatepack, cookbooks_home,
     with open(build_config) as cfg:
         cfg = json.load(cfg)
 
+    cookbook_name = cfg['name']
     cookbook = None
-    if cookbook_path:
-        try:
-            cookbook = book.CookBook(cookbook_path)
-        # change this to a more specific error
-        except ValueError:
-            # create a new one please
-            pass
+    cookbook_path = os.path.join(cookbooks_home, cookbook_name)
```

Comment:
Changing this and the signature of `def build_cookbook` above means you would need to use the "global" `--cookbooks` option to change the target location for your built cookbook. This seems pretty reasonable and simplifies things. :+1: I think this means you can/should remove the `--cookbook` argument from the `build` command too.